src/app/shared/preview-results/preview-results-table/preview-results-table.component.ts

```diff
@@ -40,13 +43,21 @@ export class PreviewResultsTableComponent implements OnChanges {
   @Input()
   public selectedDocumentId: string;
 
+  @Input()
+  public loaded: boolean;
+
   public page = 0;
 
   @Output()
   public selectDocument = new EventEmitter<DocumentModel>();
 
   public readonly pageSize = PAGE_SIZE;
 
+  public loaded$: Observable<boolean>;
+
+  constructor(private store: Store<AppState>) {
```

Comment:
These lines are probably not needed since `loaded` is passed as an input parameter.